eks.tf

```diff
@@ -0,0 +1,32 @@
+data "aws_eks_cluster" "cluster" {
+    name = module.core-infra-eks.cluster_id
+}
+
+data "aws_eks_cluster_auth" "cluster" {
+    name = module.core-infra-eks.cluster_id
+}
+
+provider "kubernetes" {
+    host = data.aws_eks_cluster.cluster.endpoint
+    cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
+    token = data.aws_eks_cluster_auth.cluster.token
+    load_config_file = false
+    version = "~> 1.9"
+}
+
+module "core-infra-eks" {
+    source = "terraform-aws-module/eks/aws"
```

Comment:
source = "git::https://github.com/terraform-aws-modules/terraform-aws-vpc.git?ref=v2.38.0"

^ that's just an example but you should be doing something like that.